wally.toml

```diff
@@ -5,6 +5,9 @@ registry = "https://github.com/UpliftGames/wally-index"
 realm = "shared"
 
 [dependencies]
+Fusion = "nottirb/fusion-beta@0.1.3-beta"
+FusionUtils = "nottirb/fusion-utils@0.1.4"
+StoryTaskBar = "nottirb/storytaskbar@0.1.4"
```

Comment:
what's a story task bar?

Reply:
Story taskbar is used to easily create Hoarcekat compatible code with easy-to-use buttons.

Reply:
Neat!